### ISSUE TITLE
[Priest] - Lightwell only allowing one charge per player

### ISSUE THREAD

**Issue by kinpie** [SunwellTracker/issues#2697]
Decription: Lightwell only allows one use per player.

How it works: After a player uses a Lightwell charge, they are not able to use the same Lightwell again.

How it should work: Players are able to use Lightwell more than once.  Upon clicking Lightwell, a player receives the healing over time for 6 seconds.  The player is able to click Lightwell again once the healing over time duration has ended.  Once 10 charges have been used, the Lightwell disappears.

Source:  
![LWbug](https://user-images.githubusercontent.com/36087298/71107555-4c28be80-218f-11ea-9d7a-81dcd1fc043d.jpg)

**Comment by Aridam** [comment]
https://us.battle.net/forums/en/wow/topic/1827669123 (Cata)
https://classic.wowhead.com/spell=724/lightwell#comments:id=2773342 (Vanilla)

Has been the same throughout. Patchnotes list no changes to this - - only to the amount of charges available.

**Comment by kinpie** [comment]
Wrong, multiple things have been changed about Lightwell.  The cooldown, spell description, healing effect, number of charges, range required, and the ability for Lightwell's effect to be canceled upon taking damage greater than or equal to 30% of your hp.

Not only is your statement about it being "the same throughout" completely false, your evidence is in part from a forum post that has no facts to support your argument.  Both of your links are irrelevant, Sunwell is Wotlk, not "(Cata)" or "(Vanilla)".

**Comment by Aridam** [comment]
>Wrong, multiple things have been changed about Lightwell. The cooldown, spell description, healing effect, number of charges, range required, and the ability for Lightwell's effect to be canceled upon taking damage greater than or equal to 30% of your hp.

Methinks somebody didn't quite read or understand what was written under the two links on my comment, and decided to be an Angry Alice instead.

>Not only is your statement about it being "the same throughout" completely false, your evidence is in part from a forum post that has no facts to support your argument. Both of your links are irrelevant, Sunwell is Wotlk, not "(Cata)" or "(Vanilla)".

Said by someone who actually contributed effectively nothing to this report.

Speaking of, here: [chill a little.](https://youtu.be/c1ZoAquHGPU?t=141)

**Comment by kinpie** [comment]
The comments in your links are IRRELEVANT!  You cannot simply say something works this way or that just because somebody says it does...  If I told you the only way to get to Northrend is by swimming, that doesn't mean it's true!

I demand that a closer look be taken into Lightwell with respect to how many charges can be used by each individual player.

**Comment by Aridam** [comment]
No demands needed, Karen. The confirmation is there, if you just check the hyperlinked words at the bottom of the comment just-above you - - which of course you didn't, obviously - - and you can then feel free to drop the "I wanna speak to the manager" attitude you've got going on right now.

**Comment by felmad** [comment]
I can cofirm that Lightwell can be used once per person, while there is no such restriction mentioned here:
https://wow.gamepedia.com/Lightwell
https://wowwiki.fandom.com/wiki/Lightwell_(old)

**Comment by felmad** [comment]
IDK what happened, but Lightwell now properly can be used again after used once. But there is probably another bug yet I did not test it well - if priest will take dmg equal to 30% of his/her HP, lightwell cannot be used - as can be read above, HoT should be removed instead of the ability to use lightwell:
"In patch 3.0, Lightwell received further changes: the healing effect only broke if damage received equaled more than 30% of total health. Furthermore the total number of charges was increased to 10."

**Comment by felmad** [comment]
PS
Above and below*
I could not edit previous post for some reason.
